scripts/dataGenerator.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs');
+const csvWriter = require('csv-write-stream');
+const faker = require('faker');
+const writer = csvWriter();
+
+const randomBetweenTen = () => {
+  return Math.ceil(Math.random() * 10);
+}
+
+const getRandomInt = (max) => {
+  return Math.floor(Math.random() * Math.floor(max));
+}
+
+const countryOfOrigin = [
+  "the United States",
+  "the United States",
+  "the United States",
+  "the United States",
+  "the United States",
+  `${faker.address.country()}`,
+];
+
+const dataGen = (num) => {
+  let id = 1;
+  writer.pipe(fs.createWriteStream('SeedData.csv'));
+  for (let i = 0; i < num; i++) {
+    for (let j = 0; j < randomBetweenTen(); j++) {
+      writer.write({
+        id: id,
+        name: `${faker.name.firstName()} ${faker.name.lastName()}`,
+        stars: getRandomInt(21),
```

Comment:
Just out of curiosity, why do stars go to 21?

Reply:
Yeah, It's just the way this program calculates the stars on the front end.